CourseGrab/SceneDelegate.swift

```diff
@@ -18,11 +18,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
         let window = UIWindow(windowScene: scene)
         let splash = UIViewController()
-        splash.view.backgroundColor = .white
+        splash.view.backgroundColor = .courseGrabBlack
         window.rootViewController = splash
 
         Auth.auth().addStateDidChangeListener { (auth, user) in
-            if let email = user?.email, email.split(separator: "@").last != "cornell.edu" {
+            if let email = user?.email, email.split(separator: "@").last != "cornell.edu" && email != "coursegrabappstore@gmail.com" {
```

Comment:
Do we still need this hard coded?

Reply:
I was talking about the coursegrabappstore email.

Reply:
My bad, I just added it back in to reflect whats currently on the app store but its probably better to take it out.